<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meshSettings_Dialog</class>
 <widget class="QDialog" name="meshSettings_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>265</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mesh Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
	background-color: rgb(43, 43, 43)
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string>Element Size</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="maxSizeInput_lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit::disabled{
	color: rgb(153, 153, 153)
}
QLineEdit{
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255)
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="auto_radioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::disabled{
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(128, 128, 128)
}</string>
        </property>
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="userDefined_radioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::disabled{
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(128, 128, 128)
}</string>
        </property>
        <property name="text">
         <string>User Defined</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>261</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>61</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="apply_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover{
background: rgb(144, 200, 246)
}
QPushButton::disabled{
color: rgb(153, 153, 153)
}
QPushButton{
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover{
background: rgb(144, 200, 246)
}
QPushButton::disabled{
color: rgb(153, 153, 153)
}
QPushButton{
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
